test/scenarios-bootc/periodics/cos9-src@optional.sh

```diff
@@ -11,12 +11,27 @@ scenario_create_vms() {
     prepare_kickstart host1 kickstart-bootc.ks.template cos9-bootc-source-optionals
     # Two nics - one for macvlan, another for ipvlan (they cannot enslave the same interface)
     launch_vm --boot_blueprint centos9-bootc --network "${VM_MULTUS_NETWORK},${VM_MULTUS_NETWORK}"
+
+    # Open the firewall ports. Other scenarios get this behavior by
+    # embedding settings in the blueprint, but there is no blueprint
+    # for this scenario. We need do this step before running the RF
+    # suite so that suite can assume it can reach all of the same
+    # ports as for any other test.
+    configure_vm_firewall host1
 }
 
 scenario_remove_vms() {
     remove_vm host1
 }
 
 scenario_run_tests() {
-    run_tests host1 suites/optional/
+        run_tests host1 \
+        --variable "PROXY_HOST:${VM_BRIDGE_IP}" \
+        --variable "PROXY_PORT:9001" \
```

Comment:
Are the proxy variables needed?

Reply:
You are right, removed them in https://github.com/openshift/microshift/pull/4825/commits/540450bb8cf61096c768e199a2559706dec0894f commit